db/migrate/20211103075455_create_measuretypes_taxonomies.rb

```diff
@@ -0,0 +1,10 @@
+class CreateMeasuretypesTaxonomies < ActiveRecord::Migration[6.1]
+  def change
+    create_table :measuretypes_taxonomies do |t|
```

Comment:
table should be called measuretype_taxonomies

Reply:
Not in Rails conventions it shouldn't. Many to many tables have plurals on both sides

Reply:
Can override the convention if you really want to 😄